packages/core/admin/server/services/project-settings.js

```diff
@@ -52,8 +49,15 @@ const parseFilesData = async (files) => {
 
 const getProjectSettings = async () => {
   const store = strapi.store({ type: 'core', name: 'admin' });
+
+  // Returns an object with file inputs names as key and null as value
+  const defaultProjectSettings = PROJECT_SETTINGS_FILE_INPUTS.reduce((prev, cur) => {
+    prev[cur] = null;
+    return prev;
+  }, {});
```

Comment:
Why don't you just store them as an object? :D

Reply:
would be a question for @nathan-pichon if you want to handle it? I don't think @vincentbpro will have the time!

Reply:
> Why don't you just store them as an object? :D

It is used everywhere as an array, and this is the only part where it is used as a hashmap. So I guess it doesn't make sense to store it as an object from the start and transform it several times into an array.